packages/webpack-cli/lib/groups/basicResolver.js

```diff
@@ -30,6 +30,9 @@ function resolveArgs(args) {
         if (arg === 'watch') {
             finalOptions.options.watch = true;
         }
+        if (arg === 'amd') {
+            finalOptions.options.amd = args[arg];
+        }
```

Comment:
I think here we can tell the user to use `--no-amd` instead. Any thoughts?

Reply:
Yes, I think we should avoid it in our code, what is webpack.cli return on amd?

Reply:
If only `false` we should support on `--no-amd`

Reply:
yes only `amd: false` is allowed.

Reply:
So let's support only `--no-amd`, do we need fix something on webpack side?

Reply:
Webpack doesn't export any negative flags. They are added by CLI only. I think we can do something like - 

```js
if (arg === 'amd') {
       if (args[arg] === true) {
           logger.error('configuration.amd can accept only false as boolean. Only --no-amd is supported.');
            process.exit(2);
        }
          finalOptions.options.amd = args[arg];
   }
```

Reply:
We should avoid put logic for special values of flags inside webpack-cli, we should fix it on webpack side

Reply:
/cc @sokra